vuepress/docs/next/docs/ecr/ecr-bundle-versions-faq.md

```diff
@@ -0,0 +1,73 @@
+# Bundle versions and updates - FAQ
+
+## Does the ECR support versioning?
+A bundle, as already discussed in the [overview](./ecr-overview.md), is a package containing one or more components.
+As in many other packaging systems, the ECR support bundle versioning, allowing the developer to create and release improvements of his package over time.
+
+## How is a bundle version defined?
+
+In the ECR, the bundle custom resource comes with a set of `tags`, each one representing a version of the bundle.
+Here an example for a CMS quickstart bundle
+```
+apiVersion: entando.org/v1
+kind: EntandoDeBundle
+metadata:
+    name: cms-quickstart-bundle
+spec:
+  details:
+    name: Entando Example CMS Bundle
+    description: Example Bundle containing CMS components for the Quickstart experience
+  tags:
+    - integrity: ''
+      shasum: ''
+      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
+      version: v0.0.1
+    - integrity: ''
+      shasum: ''
+      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
+      version: v0.0.2
+```
+
+As you can see from the code above, the bundle `cms-quickstart-bundle` has 2 versions available.
+To each version in the `tags` object must correspond a tag in the git repository provided in the `tarball` field.
+
+## What format should I use to version my bundle?
```

Comment:
This needs a lead in of some kind. 
```
How you version your bundle is up to you with some restrictions on available characters (list actual restrictions here)

Our recommended approach is to review semantic versioning with the ....
```

Reply:
We are actually requiring the version to be semver to be actually processed by the component-manager